src/Bridge/Doctrine/Orm/Filter/AbstractFilter.php

```diff
@@ -173,4 +174,29 @@ protected function extractProperties(Request $request) : array
 
         return $request->query->all();
     }
+
+    /**
+     *
+     * @param $property
+     * @param $alias
+     * @param $queryBuilder
+     *
+     * @return array
+     */
+    protected function getAssociation($property,$alias,&$queryBuilder) : array
```

Comment:
Ohh, yes, I was thinking that this one was not passed by reference when testing this, but it was another problem, ill remove that